### PR TITLE
New MW Glow

### PR DESCRIPTION
**Pull request by duckmanBR**
Helping @inexorableace with the new masterwork glow

### CODE REVIEW

**Comment by joshminnie** [comment]
@duckmanBR Do you have a screenshot to show what this ends up looking like?

**Comment by duckmanBR** [comment]
Don't know if this will work... 

https://cdn.discordapp.com/attachments/333469046785966083/397933527792091136/Capture.PNG

**Comment by duckmanBR** [comment]
Ace is still tweaking the css so please don't merge it yet.

**Comment by inexorableAce** [comment]
per some user feedback going to lighten the gradient a tad to retain energy flavor visibility. 
```
background: -moz-radial-gradient(center, ellipse cover, rgba(232,165,52,0) 1%, rgba(232,165,52,0) 39%, rgba(232,165,52,0.78) 99%, rgba(232,165,52,0.80) 100%); 
background: -webkit-radial-gradient(center, ellipse cover, rgba(232,165,52,0) 1%,rgba(232,165,52,0) 39%,rgba(232,165,52,0.78) 99%,rgba(232,165,52,0.80) 100%); 
background: radial-gradient(ellipse at center, rgba(232,165,52,0) 1%,rgba(232,165,52,0) 39%,rgba(232,165,52,0.78) 99%,rgba(232,165,52,0.80) 100%);
```

**Comment by bhollis** [comment]
Could we put this behind the energy flavor and the bottom bar?

File: src/app/inventory/dimStoreItem.scss [hunk 1]
```diff
@@ -101,16 +101,16 @@ dim-store-item {
...
-      background-color: $orange;
-      filter: blur(8px);
+      width: calc(100% - 4px);
+      height: calc(100% - 4px);
+      position: absolute;
+      left: 2px;
+      bottom: 2px;
+      background: -moz-radial-gradient(center, ellipse cover, rgba(232,165,52,0) 1%, rgba(232,165,52,0) 39%, rgba(232,165,52,0.78) 99%, rgba(232,165,52,0.80) 100%); 
+      background: -webkit-radial-gradient(center, ellipse cover, rgba(232,165,52,0) 1%,rgba(232,165,52,0) 39%,rgba(232,165,52,0.78) 99%,rgba(232,165,52,0.80) 100%); 
+      background: radial-gradient(ellipse at center, rgba(232,165,52,0) 1%,rgba(232,165,52,0) 39%,rgba(232,165,52,0.78) 99%,rgba(232,165,52,0.80) 100%);
```
Comment (bhollis) [inline src/app/inventory/dimStoreItem.scss]: There's no need to specify this three times. We use autoprefixer to automatically generate necessary prefixed versions.

Comment (inexorableAce) [reply src/app/inventory/dimStoreItem.scss]: Oh cool, wasn't sure. I used the linter and it said it was all good so I left it just in case.

**Pull request merged**